gen_cert.py

```diff
@@ -1947,16 +1949,15 @@ def _generate_v4_certificate(
         # 0 1 - italic
         # 1 0 - bold
         # 1 1 - italic and bold
-        addMapping('OpenSans-Light', 0, 0, 'OpenSans-Light')
-        addMapping('OpenSans-Light', 1, 0, 'OpenSans-Bold')
-        addMapping('SourceSansPro-Regular', 0, 0, 'SourceSansPro-Regular')
-        addMapping('SourceSansPro-Regular', 1, 0, 'SourceSansPro-Bold')
-        addMapping('SourceSansPro-Regular', 1, 1, 'SourceSansPro-BoldItalic')
+        font_string = self.template_font_name + '-' + self.template_font_type
+        addMapping(font_string, 0, 0, font_string)
+        addMapping(font_string, 1, 0, self.template_font_name+'-Italic')
+        addMapping(font_string, 1, 0, self.template_font_name+'-Bold')
+        addMapping(font_string, 1, 1, self.template_font_name+'-BoldItalic')
 
         # These are ordered by preference; cf. font_for_string() above
         self.fontlist = [
-            ('SourceSansPro-Regular', 'SourceSansPro-Regular.ttf', None),
-            ('OpenSans-Light', 'OpenSans-Light.ttf', None),
```

Comment:
Looking at this again, we may want keep the OpenSans font even if someone defines a different font as preferred.